ivy/functional/frontends/paddle/tensor/linalg.py

```diff
@@ -144,3 +144,9 @@ def bmm(x, y, transpose_x=False, transpose_y=False, name=None):
 @to_ivy_arrays_and_back
 def matrix_power(x, n, name=None):
     return ivy.matrix_power(x, n)
+
+
+@with_unsupported_dtypes({"2.4.2 and below": ("float16", "bfloat16")}, "paddle")
```

Comment:
We will specifie the supported dtypes as indicated in docs, refer to the link added in the comments below

Reply:
> We will specifie the supported dtypes as indicated in docs, refer to the link added in the comments below

Yeah sure, i didn't take keen interest on this